app/views/blocked/users/confirm_block.html.erb

```diff
@@ -19,6 +19,8 @@
       <li><%= t(".will_not.comments_on_works") %></li>
       <li><%= t(".will_not.comments_elsewhere") %></li>
     </ul>
+
+    <p><%= t(".mute_users_instead", muted_users_link: link_to(t(".muted_users_link_text"), user_muted_users_path(@user))).html_safe %></p>
```

Comment:
Rather than explicitly calling `.html_safe` on the result of the call to `t`, would you mind renaming the translation key to end in `_html` and dropping the `.html_safe` call? Here and everywhere else.

(In views, using a translation key that ends in `_html` or `.html` will [automatically handle marking as html-safe](https://guides.rubyonrails.org/i18n.html#using-safe-html-translations). That's why the `.sure_html` translation above doesn't need a call to `html_safe`, even though it does contain a `<strong>` tag.)